<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choosemusic</class>
 <widget class="QDialog" name="choosemusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>歌曲选择</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/anniu/gback);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>darksambaland</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>StarofAndromeda</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ParadigmShift</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>展示专用（仅双键）</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>简单</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>普通</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>困难</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/anniu/usave);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>129</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
